src/gqlSchema.json

```diff
@@ -13086,6 +13086,102 @@
             "deprecationReason": null
           }
         ]
+      },
+      {
+        "name": "client",
```

Comment:
Ser at dette kjem fra apollo.config.js, men du har endå ikkje svart kva den gjør.

Reply:
Jo, såg kommentaren tidligere. Litt tungvindt at du må ha graphql-api kjørende, men så lenge du fikser det seinere så får vi leve med det.

Reply:
apollo.config.js er config for vscode plugin til Apollo Graphql. Om man kjører opp graphql lokalt vil vscode hente skjema fra localhost og man får typesikkerhet, autocomplete og mulighet til hopper raskt mellom typer med ctrl-klikk